storage/diemdb/src/metrics.rs

```diff
@@ -132,3 +132,65 @@ pub(crate) static BACKUP_STATE_SNAPSHOT_LEAF_IDX: Lazy<IntGauge> = Lazy::new(||
     )
     .unwrap()
 });
+
+/// Rocksdb metrics
```

Comment:
It's against the guide line I think ("As a rule of thumb, either the sum() or the avg() over all dimensions of a given metric should be meaningful (though not necessarily useful). If it is not meaningful, split the data up into multiple metrics."), but grouping these into one single metric like the "DIEM_STORAGE_OTHER_TIMERS_SECONDS" will make the work on the dashboard side easier (make a row that's auto repeated on "property_name" and you don't need to update the dashboard when you add similar metrics.); On the reporting side you can loop across all property names as well.

 I'm leaning towards less duplication in code and dashboard. But up to you.